nano/node/cli.cpp

```diff
@@ -62,6 +62,7 @@ void nano::add_node_options (boost::program_options::options_description & descr
 	("migrate_database_lmdb_to_rocksdb", "Migrates LMDB database to RocksDB")
 	("diagnostics", "Run internal diagnostics")
 	("generate_config", boost::program_options::value<std::string> (), "Write configuration to stdout, populated with defaults suitable for this system. Pass the configuration type node, rpc or log. See also use_defaults.")
+	("update_config", boost::program_options::value<std::string> (), "Reads the current node configuration and updates it with missing keys and values and delete keys that are no longer used. Updated configuration is written to stdout.")
```

Comment:
The command takes an argument but ignores it.

Reply:
The intention was for future support for updating rpc and log configs too. However, they are quite small configs and they don't change as often the node config.
I'll remove it for now

Reply:
You can either remove it or make sure it is set to "node".